lib/connect/regions.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 const cli = require('heroku-cli-util')
+const co = require('co')
 
 const prodRegions = ['us', 'eu', 'tokyo', 'virginia', 'oregon', 'frankfurt', 'sydney']
```

Comment:
I think `dublin` should be in this list -- otherwise, we'll get an error if we get a dublin app's info from the API below.